framework/src/Degradations/PhantomCharacter.cpp

```diff
@@ -645,6 +638,8 @@ degradeComposant(cv::Mat &output,
                  ,
                  cv::Mat &degrads
 #endif //SAVE_DEGRADATIONS_IMAGE
+                 , 
+                 QString phantomPatternsPath
```

Comment:
you should pass a const reference here:
```suggestion
                 const QString &phantomPatternsPath
```